packages/cdktf/lib/tokens/token.ts

```diff
@@ -93,6 +93,27 @@ export class Token {
  * Less oft-needed functions to manipulate Tokens
  */
 export class Tokenization {
+  /**
+   * Reverse any value into a Resolvable, if possible
+   */
+  public static reverse(x: any): IResolvable[] {
+    if (Tokenization.isResolvable(x)) {
+      return [x];
+    }
+    if (typeof x === "string") {
+      return Tokenization.reverseString(x).tokens;
+    }
+    if (Array.isArray(x)) {
+      const reversedList = Tokenization.reverseList(x);
+      return reversedList ? [reversedList] : [];
+    }
+    if (typeof x === "number") {
+      const reversedNumber = Tokenization.reverseNumber(x);
+      return reversedNumber ? [reversedNumber] : [];
+    }
+    return [];
```

Comment:
Should generic object values be checked?
Seems similar to resolve flow.

Reply:
No I think this is not necessary, we traverse the object elsewhere